src/OrleansTestingHost/OrleansConfigurationForTesting.xml

```diff
@@ -4,7 +4,7 @@
     <StorageProviders>
       <Provider Type="Orleans.Storage.MemoryStorage" Name="MemoryStore" />
       <Provider Type="Orleans.Storage.MemoryStorage" Name="Default" />
-      <!--<Provider Type="Orleans.Storage.AzureTableStorage" Name="AzureStore" />-->
+      <Provider Type="Orleans.Storage.AzureTableStorage" Name="AzureStore" DataConnectionString="UseDevelopmentStorage=true"/>
```

Comment:
please do not change this in this file. This file is used by a lot of other tests. By addding this lien you now caused all other unrelated tests to load AzureStore provider, and potentially try to connect to dev storage.
Instead, for your test only, you can use programmable APIs to tweak configuration.
Look for example into ControllableStreamProviderTests.AdjustForTest.